app/uk/gov/hmrc/hmrcemailrenderer/templates/tdq/tdqFphReportNonCompliant.scala.txt

```diff
@@ -1,28 +1,28 @@
 @import uk.gov.hmrc.hmrcemailrenderer.templates.tdq.params.TdqFphReportParams
-@(params: Map[String, Any])Fraud prevention headers report for VAT (Making Tax Digital)
-@defining(TdqFphReportParams(params)) { reportParams =>
+@(params: Map[String, Any])
+@defining(TdqFphReportParams(params)) { reportParams =>Fraud prevention headers report for @reportParams.regimeLongForm
```

Comment:
Needed to move the @defining up so that reportParams is already defined. The missing space between => and Fraud is deliberate - without it the email renders with a leading space.